cmd/convox/templates/init/rails/docker-compose.yml

```diff
@@ -6,3 +6,5 @@ web:
   ports:
     - 80:4000
     - 443:4001
+  volumes:
```

Comment:
I think we should confidently switch back to volumes ASAP. 

Docker for Mac is documented and downloadable as if its in GA: https://beta.docker.com/docs/mac/getting-started/

I'm testing how fast beta keys get delivered now. Rumor is it should be fast, though it's not yet totally automatic, i've been waiting for 30m now.

Volumes also work on every other way people are running Docker.

The convox sync stuff is very experimental, and people are having real problems with it out in the wild. https://github.com/convox/rack/pull/629

So I don't see any harm in this landing ASAP. Skate to where the puck is headed.